custom_dc/custom_data.md

```diff
@@ -50,45 +50,96 @@ Statistical variables must follow a certain model; in particular, they must repr
 | San Jose | 2023 | private | middle | 200 |
 | San Jose | 2023 | private | secondary | 100 |
 
-Although the properties of school type and school level may already be defined in the Data Commons knowledge graph (or you may need to define them), they _cannot_ be present as columns in the CSV files that you store in Data Commons. Instead, you must create separate "count" variables to represent each case. In our example, you would actually need 6 different variables:
-- `Count_Public_Elementary`
-- `Count_Public_Middle`
-- `Count_Public_Secondary`
-- `Count_Private_Elementary`
-- `Count_Private_Middle`
-- `Count_Private_Secondary`
+The measure here is a simple count; the set of things is "schools"; and the constraints are the type and levels of the schools, namely "public", "private", "elementary", "middle" and "secondary". All of these things must be encoded as separate variables. Therefore, although the _properties_ of school type and school level may already be defined in the Data Commons knowledge graph (or you may need to define them), they _cannot_ be present as columns in the CSV files that you store in Data Commons. Instead, you must create separate "count" variables to represent each case. In our example, you would actually need 6 different variables:
+- `CountPublicElementary`
+- `CountPublicMiddle`
+- `CountPublicSecondary`
+- `CountPrivateElementary`
+- `CountPrivateMiddle`
+- `CountPrivateSecondary`
```

Comment:
Can we have '_' separators: `Count_Public_Elementary`

Also, if this is meant for schemaful import, the recommended dcid with populationType would be `Count_School_Public_Elementary`

Reply:
Yeah, this wasn't necessarily supposed to represent any existing entities, but sure, I will rewrite this as you suggest throughout.